tracer/test/snapshots/AwsSqsTests.NetCore.SchemaV1.verified.txt

```diff
@@ -62,7 +64,9 @@
       http.method: POST,
       http.status_code: 200,
       http.url: http://localhost:00000/,
-      span.kind: client
+      peer.service: MyAsyncSQSQueue2,
```

Comment:
On `CreateQueue` the `peer.service` gets added correctly, but not in the Producer/Consumers requests. This needs to be fixed.